docs/crud.md

```diff
@@ -0,0 +1,103 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
+```
+
+#### Response OK
+```http
+HTTP/1.1 200 OK
+Date: Mon, 27 Sept 2016 12:28:53 GMT
+Content-Length: ...
+Content-Type: application/json
+Etag: "3-6494e0ac6494e0ac"
+```
+```json
+{
+    "_id": "types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee",
```

Comment:
It's actually the other one that is wrong, see my comment [below](https://github.com/cozy/cozy-stack/pull/7#discussion_r80943692)